src/services/conseillers/conseillers.class.js

```diff
@@ -623,6 +623,19 @@ exports.Conseillers = class Conseillers extends Service {
       const estDoublon = motif === 'doublon' && nbDoublonsReel > 1;
       const aDoublonRecrute = await db.collection('conseillers').countDocuments({ 'email': email, 'statut': 'RECRUTE' });
       const tableauCandidat = await db.collection('conseillers').find(instructionSuppression).toArray();
+      let instructionSuppressionMER = motif === 'doublon' ?
+        { 'conseiller.$id': new ObjectId(id), 'conseillerObj.email': email, 'statut': {} } :
+        { 'conseillerObj.email': email, 'statut': {} };
+      instructionSuppressionMER.statut = { $in: [
```

Comment:
Ca fonctionne ? je ne sais pas si le fait de rajouter un object ici ca va passer car ca va couper entre les premiers matchs et ici non ?

Reply:
Je ne vois pas ce que tu veux dire par couper entre les premiers matchs, mais ajouter le $in après le if fonctionne